data/scenarios/Tutorials/world101.yaml

```diff
@@ -0,0 +1,78 @@
+name: First steps
+description: |
+  Learn some of the first steps you might take on the new planet.
+objectives:
+  - goal:
+      - You're ready to graduate to something a bit more complex! This
+        multi-step tutorial will walk you through a few of the first
+        steps you might take in exploring the new planet, using many
+        of the commands and techniques you have learned in the
+        previous tutorial challenges.
+      - You can see that your base starts out with some key devices
+        installed and some basic supplies for building robots.  To
+        build more advanced devices and produce more robots,
+        you'll need to explore, gather resources, and set up some
+        automated production pipelines.
+      - At this point you may want to create an external `.sw` file
+        with useful definitions you create.  You can then load it
+        via the `run` command.  See
+        https://github.com/swarm-game/swarm/tree/main/editors
```

Comment:
It's kind of annoying that because the world panel is selected, I can not actually copy or even click on this link. :confused:

Reply:
You're right, that is very annoying.  In my case the link is even highlighted (underlined) when I mouse over it, but I can't do anything with it.  I'm not sure how to fix this...

Reply:
There is https://hackage.haskell.org/package/brick-0.73/docs/Brick-Widgets-Core.html#v:hyperlink but I can't get it to work.  And in any case it would be difficult to apply it to something inside a description.